external-packages.md

```diff
@@ -0,0 +1,64 @@
+# External Packages
+
+## Bower
+
+Bower can manage components that contain HTML, CSS, JavaScript, fonts or even image files. Bower doesn’t concatenate or minify code or do anything else - it just installs the right versions of the packages you need and their dependencies.
+
+### Bower packages
+
+Bower packages can be installed from a variety of sources, including a registered bower package (a repo that has a `bower.json` file defined`), a Github shorthand (<user or org>/<repo_name>), a Github URL, or just a plain URL that points to a javascript file.
```

Comment:
`(<user or org>/<repo_name>)` does not display correctly in markdown version. it comes out as "(/)"